package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "moduleLoader",
+    "name": "@degjs/moduleLoader",
```

Comment:
NPM will yell at the name `moduleLoader`. It doesn't like capital letters in module names
https://docs.npmjs.com/files/package.json

Consider `module-loader` and updating README to match

Reply:
Good catch, thanks. Updated.